DrinkManagerWeb/Views/Drinks/DrinkDetails.cshtml

```diff
@@ -32,26 +32,26 @@
     <div class="col-12 col-md-8">
         <div class="card mb-3">
             <div class="mb-3 card-header">
-                <div class="d-flex">
-                    <h1>@Model.Name</h1>
-                    <div class="buttons d-flex align-items-center ml-auto">
+                <div class="row">
+                    <h1 class="col-12 text-center text-xl-left col-xl-6">@Model.Name</h1>
+                    <div class="buttons d-flex align-items-center ml-xl-auto col-12 col-xl-6 justify-content-center justify-content-xl-end">
                         @if (Model.IsFavourite)
                         {
-                            <a asp-controller="Drinks" asp-action="RemoveFromFavourite" asp-route-id="@Model.DrinkId" class="btn btn-primary mr-3">
+                            <a asp-controller="Drinks" asp-action="RemoveFromFavourite" asp-route-id="@Model.DrinkId" class="btn btn-primary mr-3 text-nowrap">
```

Comment:
When this button change to "Remove from favourites" it doesn't fit the screen on telephones views. :(

Reply:
Another fix added, please check now :)

Reply:
Now it's ok, thanks :)